cl/lib/decorators.py

```diff
@@ -105,7 +107,14 @@ def decorator(view_func):
         async def _wrapped_view(request, *args, **kwargs):
             url_path = urlparse(request.build_absolute_uri()).path
             hash_key = md5(url_path.encode("ascii"), usedforsecurity=False)
-            cache_key = f"custom.views.decorator.cache:{hash_key.hexdigest()}"
+            # Use S3 cache in production only
+            cache = (
+                caches["s3"] if not settings.DEVELOPMENT else caches["default"]
+            )
```

Comment:
This is interesting, but I don't like how it uses S3 without saying so in the name or parameters. This decorator should be usable without reading its code and figuring out that it uses an exotic (and slow-ish) cache. 

Maybe we default to the regular cache, but allow the user to pass through a different one if they want via a parameter?